src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Consider removing eslint-disable comment and fixing the accessibility issue by adding a label to the button. ♿

Reply:
ok